apps/nextjs-website/src/_contents/appIo/overview.ts

```diff
@@ -43,6 +44,7 @@ export const appIoOverview: OverviewData = {
     ],
   },
   tutorials: tutorials,
+  guides: guides,
```

Comment:
The term `guides` can be confusing once the guides are integrated. 
```suggestion
  postIntegration: postIntegration,
```